inference-engine/src/low_precision_transformations/src/concat_multi_channels.cpp

```diff
@@ -224,14 +228,13 @@ void ConcatMultiChannelsTransformation::fillQuantization(
                 // add concatenated dequantization operations to dequantization collection
                 dequantization.push_back(getConcatenatedDequantization(concat, dequantizationToConcatenate));
             } else {
-                std::shared_ptr<ngraph::opset1::StridedSlice> stridedSlice = ngraph::as_type_ptr<ngraph::opset1::StridedSlice>(parent);
-                if (stridedSlice) {
+                const size_t sourceOutputIdx = NetworkHelper::getParentOutputIndex(parent, layer);
+                if (parent->get_input_shape(0)[1] != parent->get_output_shape(sourceOutputIdx)[1]) {
```

Comment:
1. Check input shape, please: we can have one dimensional tensor (`Concat` parameter `axis="0"`). I think we need functional test case for this case too.
2. We should check `axis` parameter equality for all affected `Concat` operations. I think we can skip `Concat` operations with negative `axis` values. Can we do it in this PR?

Reply:
Thanks, fixed! Concats with unsupported `axis` values are not included in the subgraph now. Also I added normalization of negative axis values. So, negative `axis` values it is not problem